lib/Server.js

```diff
@@ -1627,6 +1638,10 @@ class Server {
         this.sendMessage([client], "progress", this.options.client.progress);
       }
 
+      if (this.options.client && this.options.client.reconnect) {
+        this.sendMessage([client], "reconnect", this.options.client.reconnect);
+      }
```

Comment:
We need handler in our client for this like for `progress`/etc, so developers can change value using web socket API